docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building a go binary that includes golang-fips patches
+
+## Env variables
+
+`GOEXPERIMENT=strictfipsruntime` helps ensure a fips compliant binary is build.  Note that this is functionally equivalent of
+
+```
+GO_BUILDTAGS+="goexperiment.strictfipsruntime"
+```
+
+You can export more Go flags like `GO_GCFLAGS="-N -l"` to add symbols in the final binary.
+
+## Native build - Method 1
+
+The easiest way to do is to simply clone this repo and run make. see a more comprehensive script at `scripts/build1.sh`
```

Comment:
Capitalize 's' in 'see'.

Reply:
I would also say 'apply patches and from the internal go/src directory run ./make.bash' instead of just 'run make'.

Reply:
Fixed.